apps/api/content/core/repository/instructor-repository.go

```diff
@@ -0,0 +1,30 @@
+package repository
+
+import (
+	"context"
+
+	domain "github.com/gatherloop/course-explorer-monorepo"
+	"gorm.io/gorm"
+)
+
+type instructorRepository struct {
+	db *gorm.DB
+}
+
+type InstructorRepository interface {
+	GetInstructorsList(ctx context.Context) ([]domain.Instructor, error)
+}
+
+func NewInstructorRepository(db *gorm.DB) InstructorRepository {
+	return &instructorRepository{db: db}
+}
+
+func (i *instructorRepository) GetInstructorsList(ctx context.Context) ([]domain.Instructor, error) {
+	var InstructorList []domain.Instructor
+	err := i.db.Find(&InstructorList).Error
+	if err != nil {
+		return InstructorList, err
+	}
+	return InstructorList, nil
```

Comment:
ini kayaknya bisa disingkat jadi gini ? 

```suggestion
	return InstructorList, err
```

Reply:
jadi line 25-269 cuma diganti return InstructorList gitu aja?

Reply:
25-28 maksudnya

Reply:
yap betul